File: app/src/main/java/com/asfoundation/wallet/ui/Erc681ReceiverView.java
package com.asfoundation.wallet.ui;

import com.asfoundation.wallet.entity.TransactionBuilder;

interface Erc681ReceiverView {
  String getCallingPackage();

  void startEipTransfer(TransactionBuilder transactionBuilder, Boolean isBds);

  void startApp(Throwable throwable);

  void endAnimation();

  void showLoadingAnimation();
}

File: app/src/main/java/com/asfoundation/wallet/ui/Erc681ReceiverView.kt
package com.asfoundation.wallet.ui

import com.asfoundation.wallet.entity.TransactionBuilder

internal interface Erc681ReceiverView {

  fun getCallingPackage(): String?

  fun startEipTransfer(transactionBuilder: TransactionBuilder, isBds: Boolean)

  fun startApp(throwable: Throwable)

  fun endAnimation()

  fun showLoadingAnimation()

  fun launchWebViewPayment(url: String, transaction: TransactionBuilder, type: String)
}
